packages/web-client/public/images/logos/github-logo.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18.693 18.086">
+  <defs>
+    <clipPath id="github-logo-clippath">
+      <path data-name="Path 13101"
```

Comment:
Maybe it doesn’t matter because it’s built via SVG Jar/SVGO, but I’ve run SVGs through ImageOptim (which also uses SVGO I believe) to strip these kinds of things. But it actually makes it kind of annoying to edit if styles or the like need to be changed in the future. Undecided 🤔

Reply:
Removed these, I hope we can treat svg assets as... atomic? and not bother about the internal details.